plugins/azure/containerregistry/acrContentTrustEnabled.js

```diff
@@ -0,0 +1,61 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure/');
+
+module.exports = {
+    title: 'ACR Content Trust Enabled',
+    category: 'Container Registry',
+    domain: 'Containers',
+    description: 'Ensure that content trust is enabled for Azure premium container registries.',
+    more_info: 'Content trust allows you to sign the images you push to your registry. Consumers of your images (people or systems pulling images from your registry) can configure their clients to pull only signed images.',
+    recommended_action: 'Modify your container registry and enable content trust.',
+    link: 'https://learn.microsoft.com/en-us/azure/container-registry/container-registry-content-trust#enable-registry-content-trust',
+    apis: ['registries:list'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var locations = helpers.locations(settings.govcloud);
+
+        async.each(locations.registries, function(location, rcb) {
+
+            var registries = helpers.addSource(cache, source,
+                ['registries', 'list', location]);
+
+            if (!registries) return rcb();
+
+            if (registries.err || !registries.data) {
+                helpers.addResult(results, 3,
+                    'Unable to query for container registries: ' + helpers.addError(registries), location);
+                return rcb();
+            }
+
+            if (!registries.data.length) {
+                helpers.addResult(results, 0, 'No existing container registries found', location);
+                return rcb();
+            }
+
+            var found = false;
+            for (let registry of registries.data) {
+                if (!registry.id || (registry.sku && registry.sku.tier && registry.sku.tier!= 'Premium')) continue;
```

Comment:
```suggestion
                if (!registry.id || (registry.sku && registry.sku.tier && registry.sku.tier!= 'Premium')) continue;
```
maybe add a pass result for non-premium registries that says 'is not a premium registry'